doc/_includes/tutorials/common/creating-a-project.md

```diff
@@ -18,14 +18,20 @@ If you choose to set a project organism, click on the "Select an Organism" drop-
 
 The organism names are derived from the [NCBI taxonomy database](http://www.ncbi.nlm.nih.gov/taxonomy).
 
-To lock sample modification in the new project, check the "Lock sample modification" box. This setting will only allow the sample owner to edit any sample data.
+Clicking the "Next" buttons allows you to select samples that have been added to your cart to the project.
 
-When you've finished entering the details for your new project, click on the "Create Project" button. You will be redirected to the projects list.
+If there are no samples currently in the cart a message will be displayed:
 
-IRIDA will automatically generate a numeric project identifier for your project. The project identifier is used by external tools for uploading sequencing data to IRIDA. The project identifier can be found in the ["Project Details" panel]({{ site.baseurl }}/user/user/project/index.html#viewing-project-details), or in the [projects list]({{ site.baseurl }}/user/user/project/index.html#viewing-existing-projects).
+![Samples]({{ site.baseurl }}/images/tutorials/common/projects/create-project-no-samples.png)
+
+When samples are in the cart, only samples that you have full permissions on can be added to a new project.  
+
+![Samples]({{ site.baseurl }}/images/tutorials/common/projects/create-project-samples.png)
 
-#### Creating a project from cart
+To prevent samples from being modified or copied withinthe new project, check the "Prevent modification and copying of these samples" box. This setting will only allow the sample owner to edit any sample data.
```

Comment:
Add space to `withinthe`

Reply:
Updated in 9d95b3c41be2eb1d8b56777ba1f9295fbb33ec89